src/packages/rescue/data.py

```diff
@@ -0,0 +1,18 @@
+import typing
+
+import attr
+
+from ..mark_for_deletion import MarkForDeletion
+
+
+@attr.s
+class Data:
```

Comment:
Suggestions on a better name for this are welcome, i really don't like `Data` as a class name as i tend to use it as a throw-away variable...

Reply:
[Suggestion] Well, the package is `rescue.data`, how about `RescueData`?

Reply:
That is now i imported it in the API stuff... still seems a tad bad.